dotcom-rendering/src/components/FootballMatchList.stories.tsx

```diff
@@ -136,6 +140,7 @@ const moreDays: FootballMatches = [
 export const Default = {
 	args: {
 		edition: 'UK',
+		host: 'https://www.theguardian.com',
```

Comment:
Small thing, but this is technically the "origin"[^1][^2] rather than the "host"[^3] because it includes the scheme (`https:`).

[^1]: https://developer.mozilla.org/en-US/docs/Glossary/Origin
[^2]: https://developer.mozilla.org/en-US/docs/Web/API/URL/origin
[^3]: https://developer.mozilla.org/en-US/docs/Web/API/URL/host

Reply:
I've renamed it `guardianBaseUrl` which is more accurate I think but also quite obvious what it is